styleguide/components/51-templates/molecules-overview/molecules-overview.twig

```diff
@@ -1,3 +1,17 @@
+{# --- MESSAGES --- #}
+<h3 id="styleguide-messages" class="styleguide-title">Messages</h3>
+<div class="preview-inner-container">
+  <h4 class="styleguide-title">Success message</h4>
+  <p>{% include '@status-message' with { 'icon': 'icon-checkmark', 'modifier': 'messages--status', 'text': 'Aliquam ullamcorper vehicula leo. Mauris mattis egestas hendrerit. Sed vitae aliquet mauris. Morbi at erat sed mi mollis maximus at sit amet enim. Sed quis libero eu mauris eleifend accumsan vitae semper turpis. Vestibulum nisl ipsum, ullamcorper euismod ultrices nec, tempor sit amet urna. Pellentesque bibendum a mi commodo consectetur. Aliquam nec est tellus.' } %}</p>
```

Comment:
Shouldn't be contained in a paragraph tag.
Status messages can contain paragraph tags themselves.

(same for the rest of this file)

Reply:
Fixed.